src/pages/privacy.tsx

```diff
@@ -1137,16 +1136,186 @@ function Privacy() {
+                    <div className="md:pb-12">
+                        <p>Your personal information may be processed in the United States, the country where you have applied for a job, or any other country where PostHog has team members or operations.</p>
+                        <p>PostHog may transfer, store, or process your personal information in a country outside your jurisdiction, including countries outside the European Economic Area (“EEA”), Switzerland, and the United Kingdom. If we transfer personal information from the EEA, Switzerland, or United Kingdom to a country outside it, such as the United States, we will enter into Standard Contractual Clauses (“SCCs”) approved by the EU Commission or by the UK Government,  with the data importer, or take other measures to provide an adequate level of data protection.</p>
+                        <h3 id="how-long-we-keep-applicant-personal-data">How long we keep applicant personal data</h3>
+                        <p>We will hold all the data for 24 months. Prior to that, your personal information will be deleted if:</p>
+                        <ul>
+                            <li>You delete your personal information; or</li>
+                            <li>You write to us asking us to delete your personal information.</li>
+                        </ul>
+                        <h3 id="your-rights-and-choices-as-an-applicant">Your rights and choices as an applicant</h3>
+
+                        <p>Please see the section on <SmoothScrollLink
+                            to="privacy-practices"
+                            spy={true}
+                            smooth={true}
+                            offset={-128}
+                            duration={1500}
+                            className="cursor-pointer"
+                        >Global Privacy Practices and Your Rights</SmoothScrollLink> above.</p>
+                    </div>
+                    <div>
+                        <p>If you're from the EEA, UK, or Switzerland, we handle your data based on consent, legitimate interest, or legal obligations. Your data may be processed outside your country, and we keep it for 24 months unless you ask us to delete it.</p>
+                    </div>
+
+                    <div>
+                        <h2 id="dpa">
+                            <strong>Data processing agreements</strong>
+                        </h2>
+                    </div>
+                    <div></div>
+                    <div className="md:pb-12">
+                        <p>If you need to enter into a Data Processing Agreement with us, the version you need will depend on whether you have signed up for PostHog Cloud in the US or EU. Please visit our <Link href="/dpa">world-class DPA generator</Link> and follow the instructions to have it counter-signed by PostHog.</p>
+                        <p>For the avoidance of doubt, if you use PostHog Cloud EU, no PII data is transferred to the US.</p>
```

Comment:
I think we should remove this line, as per this discussion[https://posthog.slack.com/archives/C017WDX3BFZ/p1719237602984669](url)